test/integration/release-action-integration.ts

```diff
@@ -1,46 +1,87 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
 import { expect } from 'chai'
-import { before, describe, test, after } from 'mocha'
+import { before, describe, test } from 'mocha'
 import { Environment } from '../../lib/entities/environment'
-import { Release } from '../../lib/entities/release'
+import { Release, ReleasePayload } from '../../lib/entities/release'
+import { ReleaseAction } from '../../lib/entities/release-action'
 import { Space } from '../../lib/entities/space'
 import { TestDefaults } from '../defaults'
-import { getDefaultSpace } from '../helpers'
+import { createTestSpace, initClient, initPlainClient } from '../helpers'
 import { makeLink } from '../utils'
 
-describe('ReleaseAction Api', async function () {
+describe('ReleaseAction Api', async () => {
   let testSpace: Space
   let testEnvironment: Environment
   let testRelease: Release
+  let testRelease2: Release
+
+  let testReleaseAction: ReleaseAction
+  let testReleaseAction2: ReleaseAction
 
   before(async () => {
-    testSpace = await getDefaultSpace()
+    testSpace = (await createTestSpace(initClient(), 'Release Actions')) as unknown as Space
     testEnvironment = await testSpace.getEnvironment('master')
-    testRelease = await testEnvironment.createRelease({
+
+    const contentType = await testEnvironment.createContentTypeWithId('testContentType', {
+      name: 'testContentType',
+      fields: [
+        {
+          id: 'title',
+          name: 'Title',
+          localized: false,
+          required: false,
+          type: 'Symbol',
+        },
+      ],
+    })
+    await contentType.publish()
+    await testEnvironment.createEntryWithId(
+      'testContentType',
+      TestDefaults.entry.testEntryReleasesId,
+      {
+        fields: {
+          title: { 'en-US': 'Non-localized value' },
+        },
+      }
+    )
+
+    const releasePayload: ReleasePayload = {
       title: 'Release (test)',
       entities: {
         sys: { type: 'Array' },
         items: [makeLink('Entry', TestDefaults.entry.testEntryReleasesId)],
       },
-    })
-  })
+    }
+
+    ;[testRelease, testRelease2] = await Promise.all([
```

Comment:
sorry, what is that `;`? I think I asked this question before, but this looks like some hole in javascript

Reply:
Why use it instead of const or let?

Reply:
Scope mostly, I want these two variables to be available in the test (entirely), so I initialised them before:

```ts
let testRelease: Release
let testRelease2: Release


// now I can assign/use these variables
```

Your other question: using `;` is to indicate an intention of "this is a new line of code". 
Using const/let: I would have to initialise them here, but they wouldn't be available throughout the test.


This line of code is doing:

```ts
testRelease = await ...
testRelease2 = await ...
```

([stack overflow](https://stackoverflow.com/questions/58069078/semicolon-before-square-bracket))

Reply:
Ok, but does it look like a good practice to follow this approach? I feel like that would be mentioned in the "Bad parts" section of the book about Javascript

Reply:
not a bad practice AFAIK